tools/install.py

```diff
@@ -182,8 +187,9 @@ def headers(action):
       'deps/zlib/zlib.h',
     ], 'include/node/')
 
-def run(args):
+def run(opts, args):
   global node_prefix, install_path, target_defaults, variables
+  components = set(['node', 'npm', 'headers'])
```

Comment:
This should probably be without "npm" since that's added later when you set "node_install_npm" in the config.